odbc-api/src/handles/connection.rs

```diff
@@ -231,7 +231,7 @@ impl<'c> Connection<'c> {
                 self.handle,
                 info_type,
                 &mut value as *mut u16 as Pointer,
-                0,
+                mem::size_of::<*mut u16>() as i16,
```

Comment:
0 had been fine, since the attribute does not refer to a character string.

Reply:
Have read your other message now. Would you mind writing a comment above documenting the divergent behavior of ACCESS as compared to the ODBC standard? Just so I do not get the idea to "fix" this later. I'd also prefer to `use std::mem::size_of` instead of qualifying it here.

Reply:
Sure no problem! I updated the PR again